packages/stark-build/config/build-utils.js

```diff
@@ -20,11 +20,11 @@ const ANGULAR_APP_CONFIG = {
 const DEFAULT_METADATA = {
 	TITLE: "Stark Application by @NationalBankBelgium",
 	BASE_URL: "/",
-	IS_DEV_SERVER: helpers.isWebpackDevServer(),
-	HMR: helpers.hasProcessFlag("hot"),
-	AOT: process.env.BUILD_AOT || helpers.hasNpmFlag("aot"),
-	E2E: !!process.env.BUILD_E2E,
-	WATCH: helpers.hasProcessFlag("watch"),
+	IS_DEV_SERVER: helpers.hasProcessFlag("serve"),
+	HMR: helpers.hasProcessFlag("hmr"),
+	AOT: helpers.hasProcessFlag("aot"),
+	E2E: helpers.hasProcessFlag("e2e"),
```

Comment:
I don't see anywhere in our current `package.json` the flag `--e2e`. Can you confirm it is correct ? 😊

Reply:
Indeed, the flag is not used... I kept it just in case... we should find a way to know whether the build is for e2e... but I think we can do that later